gemrb/core/Scriptable/Actor.cpp

```diff
@@ -3185,7 +3185,7 @@ bool Actor::GetSavingThrow(ieDword type, int modifier, const Effect *fx)
 		if (core->HasFeedback(FT_COMBAT) && (lastSave.prevType != type || lastSave.prevRoll != ret)) {
 			// "Save Vs Death" in all games except pst: "Save Vs. Death:"
 			String msg = core->GetString(DisplayMessage::GetStringReference(HCStrings(ieDword(HCStrings::SaveSpell) + type)));
-			msg += L" " + fmt::to_wstring(ret);
+			msg += u" " + fmt::format(u"{}", ret);
```

Comment:
there are probably a number of places where we can simply collapse the `+` into the `format` call. eg `msg += fmt::format(u" {}", ret)`.

Reply:
Yeah, I'll look again.